app/views/searches/_form.html.erb

```diff
@@ -1,6 +1,6 @@
 <form class="input-group mb-3" 
       role="search"
-      action="/search"
+      action="<%= search_action || "/search" %>"
```

Comment:
This does not work for me.

`undefined local variable or method 'search_action' for an instance of #<Class:0x000000017773d520>`

My mistake?

Reply:
Removed in the latest commit.